src/uthash.h

```diff
@@ -146,8 +158,12 @@ do {
 do {                                                                             \
   (tbl)->bloom_nbits = HASH_BLOOM;                                               \
   (tbl)->bloom_bv = (uint8_t*)uthash_malloc(HASH_BLOOM_BYTELEN);                 \
-  if (!(tbl)->bloom_bv) {                                                        \
-    uthash_fatal("out of memory");                                               \
+  if (!((tbl)->bloom_bv))  {                                                     \
+    if (HASH_NOMEM_OK) {                                                         \
+      _mem_ok = 0;                                                               \
```

Comment:
I take it that is so that one macro never changes/reads args from another macro, right?